applications/GeoMechanicsApplication/tests/1D-Consolidation_all_stages/README.md

```diff
@@ -0,0 +1,56 @@
+# 1D Consolidation
+
+This tests verifies that in a column of triangular plane strain elements, consolidation follows the solution given by Terzaghi in 1923.
+
+## Setup
+
+This test consists of 78 6-noded elements (using the SmallStrainUPwDiffOrderElement2D6N class) of b x h = 0.1 x 1.0 meter. A
+schematic representation can be found in the figure below:
+
+![MeshStructure](MeshStructure.svg)
+
+All nodes on the sides have fixed horizontal displacements. The bottom nodes have fixed displacement in both the horizontal and the vertical
+direction. At the top of the column a vertical compressive line load with a magnitude of 1.0 N/m is applied. This load is kept constant during the whole analysis time. Note that gravity is absent in this computation, the top load generates the excess pore pressure. In the first stage no Dirichlet boundaries for the water pressure D.O.F. are given, such that all water is contained within the column. In all later stages, the water pressure on the top of the column is specified to be 0. [Pa]. This creates an outflow boundary at the top.
```

Comment:
I think it would be better to also mention here that "p" in the figure is the water pressure.
```suggestion
direction. At the top of the column a vertical compressive line load with a magnitude of 1.0 N/m is applied. This load is kept constant during the whole analysis time. Note that gravity is absent in this computation, the top load generates the excess pore pressure. In the first stage no Dirichlet boundaries for the water pressure D.O.F. are given, such that all water is contained within the column. In all later stages, the water pressure on the top of the column (p) is specified to be 0 [Pa]. This creates an outflow boundary at the top.
```

Reply:
Mentioned: the water pressure (p) on the top
changed 0. to 0.0 to match the 1.0 earlier.